cupsfilters/ipp.c

```diff
@@ -333,4 +418,224 @@ get_printer_attributes3(http_t *http_printer,
   if (have_http == 0) httpClose(http_printer);
   return NULL;
 }
+const char*
+ippfind_based_uri_converter (const char *uri ){
```

Comment:
Why don't you keep the "is_fax" switch here. As you are putting this function into the API (the ipp.h file) it should be universal for both print and fax, especially with its name ippfind_based_uri_converter().
Make it
```
const char*
￼ippfind_based_uri_converter (const char *uri, int is_fax){
```
and inside switch between usinf the "rp" or the "rfo" TXT field for the resource.